<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Features_Keypoints</class>
 <widget class="QWidget" name="Features_Keypoints">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>236</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Keypoints Estimation</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="4" column="0">
       <widget class="QRadioButton" name="radioButton_Curvature">
        <property name="text">
         <string>Curvature</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="spinBox_BRISK">
        <property name="minimum">
         <number>2</number>
        </property>
        <property name="value">
         <number>30</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="radioButton_AGAST">
        <property name="text">
         <string>AGAST</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QDoubleSpinBox" name="doubleSpinBox_Curvature">
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="radioButton_BRISKQuad">
        <property name="text">
         <string>BRISKQuad</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="spinBox_AGAST">
        <property name="minimum">
         <number>2</number>
        </property>
        <property name="value">
         <number>30</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="radioButton_BRISK">
        <property name="text">
         <string>BRISK</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QRadioButton" name="radioButton_SIFT3D">
        <property name="text">
         <string>SIFT3D</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QPushButton" name="pushButton_Compute">
        <property name="text">
         <string>Compute Keypoints</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>radioButton_BRISK</tabstop>
  <tabstop>radioButton_AGAST</tabstop>
  <tabstop>radioButton_SIFT3D</tabstop>
  <tabstop>spinBox_BRISK</tabstop>
  <tabstop>spinBox_AGAST</tabstop>
  <tabstop>pushButton_Compute</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
